apps/filebrowser/src/filebrowser/conf.py

```diff
@@ -40,6 +40,18 @@
   default=None,
   type=str)
 
+FILE_UPLOAD_CHUNK_SIZE = Config(
```

Comment:
Is this the same for all filesystems now?

Reply:
Yes, its the same when using chunked uploader.